client/plugins/init.client.js

```diff
@@ -77,6 +77,7 @@ Vue.prototype.$sanitizeFilename = (filename, colonReplacement = ' - ') => {
     .replace(lineBreaks, replacement)
     .replace(windowsReservedRe, replacement)
     .replace(windowsTrailingRe, replacement)
+    .replace(/\s+/g, ' ') // Replace consecutive spaces with a single space
```

Comment:
I noticed that titles like `foo: bar` were being turned into `foo -  bar` (with two spaces between `-` and `bar`). I figured that this is undesirable in all cases so this turns multiple spaces into one